.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+- repo: https://github.com/asottile/add-trailing-comma
+  rev: v0.6.4
+  hooks:
+  - id: add-trailing-comma
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v1.3.0
+  hooks:
+  - id: trailing-whitespace
+    args: ['--markdown-linebreak-ext=md,rst']
+  - id: double-quote-string-fixer
+  - id: end-of-file-fixer
+  - id: flake8
+    # Need duplicated config due to known issues with flake8
+    # https://github.com/pre-commit/pre-commit-hooks/issues/112
```

Comment:
note that that issue was (as far as I can tell) just about _exclude_

Reply:
meaning: you shouldn't actually need to duplicate anything and can probably get away with the default here